index.bs

```diff
@@ -1292,6 +1321,9 @@ controls the maximum [=map/size=] of a [=trigger spec map=] for an
 the maximum [=string/length=] of an [=attribution trigger=]'s [=attribution trigger/trigger context ID=].
 Its value is 64.
 
+<dfn>Destination per day rate-limit window</dfn> is a positive [=duration=] that
```

Comment:
This is a bit strange given the name of the limit already encodes day.

One option is to just not make this paramterizable, or potentially consider a different naming although I agree it's hard to come up with a name.

One potential idea would be to instead include this in [=Max destinations per rate-limit window=] by extending the tuple to include a third integer, and ensuring a different rate limit window for the new value. wdyt?

Reply:
I would lean towards not making this parametrizable. Including this in the existing tuple may be a bit confusing as they have different rate limit windows.